pkg/boot/grub/grub.go

```diff
@@ -298,14 +317,119 @@ func (c *parser) append(ctx context.Context, config string) error {
 		arg := kv[1]
 
 		switch directive {
+		case "search.file", "search.fs_label", "search.fs_uuid":
```

Comment:
Okay - that looks like Ryan's grub search branch. Might be my fault. Can you rebase this on top of the u-root master?

Reply:
The last 4 comments are in here, because I rebased on master. The PR is on your branch which is not in sync with master/main branch.